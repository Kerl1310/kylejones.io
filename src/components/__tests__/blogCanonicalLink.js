import React from 'react';
import renderer from 'react-test-renderer';

import BlogCanonicalLink from '../blogCanonicalLink';

const testUrl = "https://example.com/"

describe('Blog Canonical Link', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<BlogCanonicalLink canonicalUrl={testUrl}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
