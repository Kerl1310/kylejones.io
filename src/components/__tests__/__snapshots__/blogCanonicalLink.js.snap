// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog Canonical Link renders correctly 1`] = `
<div>
  <p>
    Originally published at
     
    <a
      href="https://example.com/"
      rel="noopener noreferrer"
      target="__blank"
    >
      https://example.com/
    </a>
  </p>
  <hr />
</div>
`;
